packages/integration-tests/tests/config.ts

```diff
@@ -0,0 +1,25 @@
+import {Page} from '@playwright/test'
+
+type TestingMode = 'production' | 'local'
+
+export class Config {
+  readonly testingMode: TestingMode
+  readonly rootUrlPath: string
+
+  constructor(env = process.env) {
+    this.testingMode = env.INTEGRATION_TESTING_MODE === 'production' ? 'production' : 'local'
```

Comment:
@mattkrick mentioned that there's an `env` variable `host` - we should use that here.

Reply:
perfect!

Reply:
updates made with [`160bed1` (#5961)](https://github.com/ParabolInc/parabol/pull/5961/commits/160bed146a1c2e275b052b98f60af4aa2592c17c)